diff -urN work.orig/extension/Makefile.am work/extension/Makefile.am
--- work.orig/extension/Makefile.am	2018-05-26 01:59:41.000000000 +0100
+++ work/extension/Makefile.am	2018-05-26 02:01:11.000000000 +0100
@@ -148,6 +148,3 @@
 	filefuncs.3am fnmatch.3am fork.3am inplace.3am \
 	ordchr.3am readdir.3am readfile.3am revoutput.3am \
 	revtwoway.3am rwarray.3am time.3am
-
-# gettext requires this
-SUBDIRS =
