--- test/Makefile.am	2020-04-14 13:07:50.627041178 -0400
+++ test/Makefile.am.new	2020-04-14 13:07:45.111018581 -0400
@@ -632,8 +632,6 @@
 	litoct.awk \
 	litoct.in \
 	litoct.ok \
-	localenl.ok \
-	localenl.sh \
 	longsub.awk \
 	longsub.in \
 	longsub.ok \
@@ -1377,7 +1375,7 @@
 	zero2 zeroe0 zeroflag
 
 UNIX_TESTS = \
-	fflush getlnhd localenl pid pipeio1 pipeio2 poundbang rtlen rtlen01 \
+	fflush getlnhd pid pipeio1 pipeio2 poundbang rtlen rtlen01 \
 	space strftlng
 
 GAWK_EXT_TESTS = \
@@ -1477,7 +1475,7 @@
 NEED_TRADITIONAL = litoct tradanch rscompat
 
 # Lists of tests that run a shell script
-RUN_SHELL = exit fflush localenl next randtest rtlen rtlen01
+RUN_SHELL = exit fflush next randtest rtlen rtlen01
 
 # List of the tests which fail with EXIT CODE 1
 FAIL_CODE1 = \
@@ -1529,7 +1527,7 @@
 	aasort aasorti arraysort asort asorti backbigs1 backsmalls1 \
 	backsmalls2 beginfile1 beginfile2 charasbytes clos1way6 concat4 \
 	dfamb1 double1 double2 errno fmttest forcenum getlndir gsubtst5 \
-	ignrcas2 lc_num1 localenl localenl mbfw1 mbprintf1 mbprintf2 \
+	ignrcas2 lc_num1 mbfw1 mbprintf1 mbprintf2 \
 	mbprintf3 mbprintf4 mbprintf5 mbstr1 mbstr2 mtchi18n nlstringtest \
 	nofile nonfatal2 numrange printhuge profile5 profile5 rebt8b2 \
 	regrange reint2 rri1 sigpipe1 sort1 sortfor sortu space sprintfc \
@@ -2454,7 +2452,7 @@
 		diff -c "$(srcdir)"/$${base}.ok  $$i ; \
 		fi ; \
 		fi ; \
-	done | more
+	done
 
 # make things easier for z/OS
 zos-diffout:
