--- test/Makefile.in	2020-04-14 07:44:52.000000000 -0400
+++ test/Makefile.in.new	2020-04-14 13:26:58.696981987 -0400
@@ -893,8 +893,6 @@
 	litoct.awk \
 	litoct.in \
 	litoct.ok \
-	localenl.ok \
-	localenl.sh \
 	longsub.awk \
 	longsub.in \
 	longsub.ok \
@@ -1638,7 +1636,7 @@
 	zero2 zeroe0 zeroflag
 
 UNIX_TESTS = \
-	fflush getlnhd localenl pid pipeio1 pipeio2 poundbang rtlen rtlen01 \
+	fflush getlnhd pid pipeio1 pipeio2 poundbang rtlen rtlen01 \
 	space strftlng
 
 GAWK_EXT_TESTS = \
@@ -1738,7 +1736,7 @@
 NEED_TRADITIONAL = litoct tradanch rscompat
 
 # Lists of tests that run a shell script
-RUN_SHELL = exit fflush localenl next randtest rtlen rtlen01
+RUN_SHELL = exit fflush next randtest rtlen rtlen01
 
 # List of the tests which fail with EXIT CODE 1
 FAIL_CODE1 = \
@@ -1793,7 +1791,7 @@
 	aasort aasorti arraysort asort asorti backbigs1 backsmalls1 \
 	backsmalls2 beginfile1 beginfile2 charasbytes clos1way6 concat4 \
 	dfamb1 double1 double2 errno fmttest forcenum getlndir gsubtst5 \
-	ignrcas2 lc_num1 localenl localenl mbfw1 mbprintf1 mbprintf2 \
+	ignrcas2 lc_num1 mbfw1 mbprintf1 mbprintf2 \
 	mbprintf3 mbprintf4 mbprintf5 mbstr1 mbstr2 mtchi18n nlstringtest \
 	nofile nonfatal2 numrange printhuge profile5 profile5 rebt8b2 \
 	regrange reint2 rri1 sigpipe1 sort1 sortfor sortu space sprintfc \
@@ -4116,11 +4114,6 @@
 	@AWKPATH="$(srcdir)" $(AWK) -f $@.awk  >_$@ 2>&1 || echo EXIT CODE: $$? >>_$@
 	@-$(CMP) "$(srcdir)"/$@.ok _$@ && rm -f _$@
 
-localenl:
-	@echo $@
-	@-$(LOCALES) AWK="$(AWKPROG)" "$(srcdir)"/$@.sh  > _$@ 2>&1 || echo EXIT CODE: $$? >>_$@
-	@-$(CMP) "$(srcdir)"/$@.ok _$@ && rm -f _$@
-
 rtlen:
 	@echo $@
 	@-$(LOCALES) AWK="$(AWKPROG)" "$(srcdir)"/$@.sh  > _$@ 2>&1 || echo EXIT CODE: $$? >>_$@
